src/libtimed/models.py

```diff
@@ -283,8 +283,8 @@ def stop(self):
         if self.current:
             attributes = self.current["attributes"]
             relationships = self.current["relationships"]
-            attributes["to-time"] = datetime.now()
-            r = self.patch({"id": self.current["id"]}, attributes, relationships)
+            attributes["to-time"] = datetime.now().strftime("%H:%M:%S")
```

Comment:
the Time transform already does that
```suggestion
            attributes["to-time"] = datetime.now()
```

Reply:
that doesn't work.

Reply:
or does it?

Reply:
yes.